tests/conftest.py

```diff
@@ -0,0 +1,40 @@
+"""Ruff is forcing me to write a docstring for conftest.py."""
+
+from pathlib import Path
+
+import pytest
+from jinja2 import Environment, FileSystemLoader
+from ruamel.yaml import YAML
+
+COPIER_CONFIG_PATH = Path(__file__).parents[1] / "copier.yml"
+INCLUDES_PATH = Path(__file__).parents[1] / "includes"
+
+def _load_copier_config() -> dict:
+    yaml = YAML(typ="safe")
+    with COPIER_CONFIG_PATH.open("r") as yfile:
+        return yaml.load(yfile)
+
+
+# don't mutate or else i'll have to get out the spray bottle (make it a fixture)
```

Comment:
Indeed, why not make it a fixture? 🙂

Reply:
just because it can get pretty baffling pretty fast when you want to use the output of a fixture to parameterize a test function or another fixture in this case. like i wanted to be able to construct the fixture params outside of a fixture body... although now that i'm thinking about it again i might be able to rig this up with indirect parameterization... but too sleepy now. 

we could also just "do both," i had used those allcaps consts to leave open the name for a fixture if we wanted one, and there's no reason that we couldn't also make a fixture for tests that wanted to request and use the whole config instead of using it to make another fixture from its fields.

Reply:
aka no good reason for it not to be one, i chose peace instead of doing battle with pytest on this night, but may not on another

Reply:
Totally fair, it gets tricky quickly with multiple levels of indirection.